backend/gradle.lockfile

```diff
@@ -0,0 +1,206 @@
+io.netty:netty-transport-native-kqueue:4.1.79.Final=compileClasspath,runtimeClasspath
+io.netty:netty-transport-native-unix-common:4.1.79.Final=compileClasspath,runtimeClasspath
+io.netty:netty-transport:4.1.79.Final=compileClasspath,runtimeClasspath
+io.projectreactor.addons:reactor-extra:3.4.8=compileClasspath,runtimeClasspath
+io.projectreactor.netty:reactor-netty-core:1.0.22=compileClasspath,runtimeClasspath
+io.projectreactor.netty:reactor-netty-http:1.0.22=compileClasspath,runtimeClasspath
+io.projectreactor:reactor-core:3.4.22=compileClasspath,runtimeClasspath
+io.zipkin.brave:brave-instrumentation-http:5.13.2=runtimeClasspath
+io.zipkin.brave:brave:5.13.2=runtimeClasspath
+io.zipkin.reporter2:zipkin-reporter-brave:2.16.3=runtimeClasspath
+io.zipkin.reporter2:zipkin-reporter:2.16.1=runtimeClasspath
+io.zipkin.zipkin2:zipkin:2.23.0=runtimeClasspath
+jakarta.activation:jakarta.activation-api:1.2.2=compileClasspath,runtimeClasspath
+jakarta.annotation:jakarta.annotation-api:1.3.5=compileClasspath,runtimeClasspath
+jakarta.persistence:jakarta.persistence-api:2.2.3=compileClasspath,runtimeClasspath
+jakarta.transaction:jakarta.transaction-api:1.3.3=compileClasspath,runtimeClasspath
+jakarta.validation:jakarta.validation-api:2.0.2=compileClasspath,runtimeClasspath
+jakarta.xml.bind:jakarta.xml.bind-api:2.3.3=compileClasspath,runtimeClasspath
+javax.activation:javax.activation-api:1.2.0=compileClasspath,runtimeClasspath
+javax.transaction:javax.transaction-api:1.3=compileClasspath,runtimeClasspath
+javax.validation:validation-api:2.0.1.Final=compileClasspath,runtimeClasspath
+javax.xml.bind:jaxb-api:2.3.1=compileClasspath,runtimeClasspath
+net.bytebuddy:byte-buddy:1.12.13=compileClasspath,runtimeClasspath
+net.minidev:accessors-smart:2.4.8=compileClasspath,runtimeClasspath
+net.minidev:json-smart:2.4.8=compileClasspath,runtimeClasspath
+org.antlr:antlr4-runtime:4.9.3=runtimeClasspath
+org.apache.commons:commons-lang3:3.12.0=compileClasspath,runtimeClasspath
+org.apache.httpcomponents:httpclient:4.5.13=compileClasspath,runtimeClasspath
+org.apache.httpcomponents:httpcore:4.4.15=compileClasspath,runtimeClasspath
+org.apache.httpcomponents:httpmime:4.5.13=runtimeClasspath
+org.apache.logging.log4j:log4j-api:2.17.0=compileClasspath,runtimeClasspath
+org.apache.logging.log4j:log4j-to-slf4j:2.17.0=compileClasspath,runtimeClasspath
+org.apache.tomcat.embed:tomcat-embed-core:9.0.65=compileClasspath,runtimeClasspath
+org.apache.tomcat.embed:tomcat-embed-el:9.0.65=compileClasspath,runtimeClasspath
+org.apache.tomcat.embed:tomcat-embed-websocket:9.0.65=compileClasspath,runtimeClasspath
+org.aspectj:aspectjweaver:1.9.7=compileClasspath,runtimeClasspath
+org.attoparser:attoparser:2.0.5.RELEASE=compileClasspath,runtimeClasspath
+org.bouncycastle:bcpkix-jdk15on:1.68=compileClasspath
+org.bouncycastle:bcpkix-jdk15on:1.70=runtimeClasspath
+org.bouncycastle:bcprov-jdk15on:1.68=compileClasspath
+org.bouncycastle:bcprov-jdk15on:1.70=runtimeClasspath
```

Comment:
org.bouncycastle:bcpkix-jdk15on:1.68=compileClasspath
org.bouncycastle:bcpkix-jdk15on:1.70=runtimeClasspath
org.bouncycastle:bcprov-jdk15on:1.68=compileClasspath
org.bouncycastle:bcprov-jdk15on:1.70=runtimeClasspath

bcprov-jdk15on:1.68 has an encrytion vulnerability.  The runtimClasspath of 1.70 may mitigate that, but I don't see a reason for the different complieClasspath/runtimeClasspath for these.

Reply:
Here is how bcpkix-jdk15on:1.68 is introduced
```
+--- org.springframework.cloud:spring-cloud-starter-openfeign -> 3.1.3
|    +--- org.springframework.cloud:spring-cloud-starter:3.1.3
|    |    +--- org.springframework.boot:spring-boot-starter:2.6.8 -> 2.7.3 (*)
|    |    +--- org.springframework.cloud:spring-cloud-context:3.1.3
|    |    |    \--- org.springframework.security:spring-security-crypto:5.6.5 -> 5.7.3
|    |    +--- org.springframework.cloud:spring-cloud-commons:3.1.3
|    |    |    \--- org.springframework.security:spring-security-crypto:5.6.5 -> 5.7.3
|    |    \--- org.springframework.security:spring-security-rsa:1.0.10.RELEASE
|    |         \--- org.bouncycastle:bcpkix-jdk15on:1.68
|    |              \--- org.bouncycastle:bcprov-jdk15on:1.68
```

this file is generated and I am not sure what the ask is here, do you want me to pin version a complie time version of `org.bouncycastle:bcprov-jdk15on:1.70` ?

Reply:
Since we're using 1.70 at run time, let's pin that to eliminate any inconsistencies.

Thanks!

Reply:
I am not sure if that makes sense for this PR though, this has not been changed from what's on main

https://github.com/CDCgov/prime-simplereport/blob/cab18b06395359518381e118ef19b3d98bd75093/backend/gradle/dependency-locks/compileClasspath.lockfile#L126

https://github.com/CDCgov/prime-simplereport/blob/cab18b06395359518381e118ef19b3d98bd75093/backend/gradle/dependency-locks/runtimeClasspath.lockfile#L137